tests/pgshovel/streams/sequences.py

```diff
@@ -1,119 +0,0 @@
-import uuid
```

Comment:
Actually looking at this now the functionality these tests are testing moved to the `replication.validation.consumers` module, so I should actually bring this file back.